### ISSUE TITLE
Bug report: issue compiling CyberChef

### ISSUE THREAD

**Issue by sssseossss** [gchq/CyberChef#1166]
**Describe the bug**
when running npm install I get the following:
> cyberchef@9.24.3 postinstall
> npx grunt exec:fixCryptoApiImports

Running "exec:fixCryptoApiImports" (exec) task
>> /bin/sh: 1: [[: not found

I checked and confirmed that I have bash as the shell. Trying it on latest Kali.

**To Reproduce**
Steps to reproduce the behaviour or a link to the recipe / input used to cause the bug:

1. follow the exact installation steps on latest Kali OS. Checked on zsh and bash as default shell for both, root and kali user.

**Expected behaviour**

**Screenshots**

**Desktop (if relevant, please complete the following information):**
 - OS: [e.g. Windows] 
 - Browser: [e.g. chrome 72, firefox 60]
 - CyberChef version: [e.g. 9.7.14]

**Additional context**

**Comment by jimmybrancaccio** [comment]
I can confirm the same issue though using `node:10-buster`. It looks like [grunt-exec](https://www.npmjs.com/package/grunt-exec) uses `/bin/sh` by default.

* https://unix.stackexchange.com/questions/155838/shell-script-throws-a-not-found-error-when-run-from-a-sh-file-but-if-entered-ma/155881
* https://stackoverflow.com/questions/12230690/string-comparison-in-bash-not-found/17753098

**Comment by spectre013** [comment]
Running into the same issue on MAC
```
100 verbose node v15.12.0
101 verbose npm  v7.6.3
102 error code 127
103 error path /Users/<username>/opt/CyberChef
104 error command failed
105 error command sh -c npx grunt exec:fixCryptoApiImports
106 verbose exit 127
```

**Comment by mt3571** [comment]
It might be worth trying against Node v10, as that is the latest version [CyberChef supports](https://github.com/gchq/CyberChef#nodejs-support)

**Comment by alexis-evelyn** [comment]
The problem is, grunt wants to use /bin/sh, here's how to fix the issue for *nix users.

In your Gruntfile.js file, add the below code to the `fixCryptoApiImports` section of your file.

```javascript
options: {
  shell: "/bin/bash"
},
```

![Screenshot_20211217_171850](https://user-images.githubusercontent.com/6305346/146614411-87aefc1f-d3eb-4c7c-9362-135c22d892af.png)

**Comment by alexis-evelyn** [comment]
> It might be worth trying against Node v10, as that is the latest version [CyberChef supports](https://github.com/gchq/CyberChef#nodejs-support)

Yep, v10 is a must for CyberChef. I installed node via nvm to allow the latest LTS while also supporting CyberChef and before I remembered to set to v10 for CC, I tried loading up the dev version. It incorrectly gave errors about my browsers being unsupported. Using v10, CC loads just fine.
